circuits/kimchi/src/nolookup/constraints.rs

```diff
@@ -31,6 +31,33 @@ pub const ZK_ROWS: u64 = 3;
 // ConstraintSystem
 //
 
+#[serde_as]
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct LookupConstraintSystem<F: FftField> {
```

Comment:
should this live in `lookup.rs`?

Reply:
It's a part of the constraint system, so I'm inclined to say it should stay here, unless we're planning to split out all of the polys for the other gates too.

Reply:
(btw this struct doesn't have documentation)